frontend/js/src/user/stats/components/UserArtistActivity.tsx

```diff
@@ -0,0 +1,196 @@
+                style={{ marginRight: 20 }}
+              />
+            </a>
+          </h4>
+        </div>
+      </div>
+      <Loader isLoading={loading}>
+        {hasError ? (
+          <div
+            style={{
+              display: "flex",
+              alignItems: "center",
+              justifyContent: "center",
+              minHeight: "inherit",
+            }}
+          >
+            <span style={{ fontSize: 24 }}>
+              <FontAwesomeIcon icon={faExclamationCircle as IconProp} />{" "}
+              {errorMessage}
+            </span>
+          </div>
+        ) : (
+          <div className="row">
+            <div className="col-xs-12">
+              <div
+                style={{ width: "100%", height: "600px", minHeight: "400px" }}
+              >
+                <ResponsiveBar
+                  data={chartData}
+                  keys={Array.from(
+                    new Set(
+                      chartData.flatMap((item) =>
+                        Object.keys(item).filter((key) => key !== "label")
+                      )
+                    )
+                  )}
+                  indexBy="label"
+                  margin={{ top: 20, right: 80, bottom: 80, left: 80 }}
+                  padding={0.2}
+                  layout="vertical"
+                  colors={{ scheme: "nivo" }}
+                  borderColor={{ from: "color", modifiers: [["darker", 1.6]] }}
+                  enableLabel={false}
+                  axisBottom={{
+                    tickSize: 5,
+                    tickPadding: 5,
+                    tickRotation: -45,
+                    renderTick: (tick) => (
+                      <g transform={`translate(${tick.x},${tick.y})`}>
+                        {tick.value
+                          .split("\n")
+                          .map((line: string, i: number) => (
+                            <text
+                              key={line}
+                              x={0}
+                              y={10 + i * 15}
+                              textAnchor="end"
+                              dominantBaseline="middle"
+                              style={{
+                                fontSize: 10,
+                                fill: "#000",
+                                transform: `rotate(-45deg)`,
+                              }}
+                            >
+                              {line}
+                            </text>
+                          ))}
+                      </g>
+                    ),
+                  }}
+                  onClick={(barData, event) => {
+                    const albumName = barData.id;
+                    const artistName = barData.indexValue;
+                    const releaseMbid =
+                      albumRedirectMapping[`${artistName}-${albumName}`];
+                    if (releaseMbid) {
+                      window.location.href = `/album/${releaseMbid}`;
+                    }
```

Comment:
Since we're using react router, you should use [useNavigate](https://reactrouter.com/6.30.0/hooks/use-navigate#usenavigate).
```suggestion
                      navigate(`/album/${releaseMbid}`);
```